client/src/components/Feed/PostCard.js

```diff
@@ -88,6 +88,7 @@ const PostCard = styled(Card)`
         margin-top: -1.8rem;
 
         .status-icon {
+          width: 1rem;
```

Comment:
The size of the blue button next to the country name is too big atm but this should fix it: 
```suggestion
          width: fit-content;
```